README.md

```diff
@@ -1,18 +1,27 @@
 # jool-docker
 [jool](https://www.jool.mx/en/index.html) as docker container.
 
-### Install
+
+### Dependencies
 
 Since jool needs a kernel module we need to prepare the host system. For this we need to meet following prerequisites on the host:
 - curl
 - docker
 - dkms
 - (GNU)tar
 - cloned [repository](https://github.com/Jasper-Ben/jool-docker)
+### Setup
+`git clone ttps://github.com/FreifunkMD/jool-docker /opt/jool-docker`
```

Comment:
Bessere Schreibweise hier:
```bash
git clone ttps://github.com/FreifunkMD/jool-docker /opt/jool-docker
/opt/jool-docker
vi install.sh
./install.sh
```

Reply:
Natürlich wird das gerendert....
also: anstatt jede Zeile in backticks zu setzen, kann eine Zeile mit ```bash beginnen, dann kommen die Zeilen roh so, wie sie gequotet werden sollen, dann folgt eine Zeile mit drei Backticks ```

Reply:
Geändert